frame/vesting/src/lib.rs

```diff
@@ -206,6 +216,40 @@ decl_module! {
 			ensure_signed(origin)?;
 			Self::update_lock(T::Lookup::lookup(target)?)
 		}
+
+		/// Create a vested transfer. 
+		///
+		/// The dispatch origin for this call must be _Signed_.
+		///
+		/// - `target`: The account that should be transferred the vested funds.
+		/// - `amount`: The amount of funds to transfer and will be vested.
+		/// - `schedule`: The vesting schedule attached to the transfer.
+		///
+		/// Emits `VestingCreated`.
+		///
+		/// # <weight>
+		/// - Creates a new storage entry, but is protected by a minimum transfer
+		///	   amount needed to succeed.
+		/// # </weight>
+		#[weight = SimpleDispatchInfo::FixedNormal(1_000_000)]
+		pub fn vested_transfer(
+			origin,
+			target: <T::Lookup as StaticLookup>::Source,
+			schedule: VestingInfo<BalanceOf<T>, T::BlockNumber>
+		) -> DispatchResult {
+			let transactor = ensure_signed(origin)?;
+			let who = T::Lookup::lookup(target)?;
```

Comment:
```suggestion
			ensure!(schedule.locked >= T::VestingDeposit::get(), Error::<T>::AmountLow);
```

Just some small optimization ;)